packages/mermaid/src/diagrams/git/gitGraphRenderer.js

```diff
@@ -87,9 +110,23 @@ const drawCommits = (svg, commits, modifyGraph) => {
   const sortedKeys = keys.sort((a, b) => {
     return commits[a].seq - commits[b].seq;
   });
+
+  const isParallelCommits = gitGraphConfig.parallelCommits;
   sortedKeys.forEach((key) => {
     const commit = commits[key];
 
+    if (isParallelCommits) {
+      if (!commit.parents.length) {
+        pos = 0;
+        if (dir === 'TB') {
+          pos = 30;
+        }
+      } else {
+        const closestParent = findClosestParent(commit.parents);
+        pos = dir === 'TB' ? commitPos[closestParent].y + 40 : commitPos[closestParent].x + 40;
```

Comment:
If we move the magic numbers 40 and 10 to constants, that would be nice.

Reply:
Noticed the `pos + 10` was very repetitive throughout the code so I also made that into a const. Some of the 10 values seemed to have a different context so I left them untouched.